src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryTable.cs

```diff
@@ -136,6 +136,90 @@ public static FormulaValue CountRows(IRContext irContext, TableValue[] args)
             return new NumberValue(irContext, count);
         }
 
+        // Count
+        public static FormulaValue Count(IRContext irContext, TableValue[] args)
+        {
+            var arg0 = args[0];
+            var count = 0;
+
+            var errors = new List<ErrorValue>();
+
+            foreach (var row in arg0.Rows)
+            {
+                if (row.Value is null)
```

Comment:
`if (row.IsBlank) { continue; }... if (row.IsError) { errors.Add(row.Error); continue; }

Reply:
The test case **Count([1, 2, 1/0, 4, 5])**, row **1/0** is giving me the following results:

- Blank: null
- Error: null
- IsBlank: false
- IsError: false
- IsValue: true
- Value: ErrorType

Is this a valid value?

Reply:
Please let me know so I can apply the suggested changes.

Reply:
Can you clarify where (in which file) you see that object? Typically we shouldn't have a DValue<T> with IsValue = true, but where the Value is an error value